test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -639,32 +637,39 @@ public interface InstructionUI {
     }
 
 
-    private static final class TimeoutHandler implements ActionListener {
+    private static final class TimeoutHandlerPanel
+            extends JPanel
+            implements ActionListener {
+
+        private static final String PAUSE_BUTTON_LABEL = "Pause";
+        private static final String RESUME_BUTTON_LABEL = "Resume";
 
-        private static final String PAUSED_LABEL_SUFFIX = " PAUSED";
-        private static final String PAUSE_BUTTON_LABEL = "Pause Timer";
-        private static final String RESUME_BUTTON_LABEL = "Resume Timer";
         private long endTime;
-        private long pauseTimeLeft = 0L;
+        private long pauseTimeLeft;
 
         private final Timer timer;
 
         private final JLabel label;
         private final JButton button;
 
-        public TimeoutHandler(final JLabel label,
-                              final JButton button,
-                              final long testTimeOut) {
-            endTime = System.currentTimeMillis() + TimeUnit.MINUTES.toMillis(testTimeOut);
+        public TimeoutHandlerPanel(final long testTimeOut) {
+            endTime = System.currentTimeMillis()
+                    + TimeUnit.MINUTES.toMillis(testTimeOut);
+
+            label =  new JLabel("", JLabel.CENTER);
+            button = new JButton(TimeoutHandlerPanel.PAUSE_BUTTON_LABEL);
```

Comment:
```suggestion
            button = new JButton(PAUSE_BUTTON_LABEL);
```
I believe it can be used directly as in `pauseToggle`.